components/tools/OmeroPy/src/omero/plugins/prefs.py

```diff
@@ -264,13 +267,18 @@ def get(self, args, config):
             for k in config.IGNORE:
                 k in keys and keys.remove(k)
 
+        hide_password = 'hide_password' in args and args.hide_password
+        is_password = lambda x: x.endswith('.pass') or x.endswith('.password')
         for k in keys:
             if k not in orig:
                 continue
             if args.KEY and len(args.KEY) == 1:
                 self.ctx.out(config[k])
             else:
-                self.ctx.out("%s=%s" % (k, config[k]))
+                if (hide_password and is_password(k)):
+                    self.ctx.out("%s=%s" % (k, '*' * len(config[k])))
```

Comment:
This still reveals the password length. OTOH it's useful to know if the field is empty.